includes/class-wc-siftscience-element.php

```diff
@@ -54,10 +52,10 @@ public function __construct( WC_SiftScience_Logger $logger ) {
 		 * desc_tip Mixed [string]
 		 *     field type of checkbox; the desc_tip text is going underneath control
 		 *     field type of select, number or text; a question mark pop-up appears before control with desc_tip text
-		 * desc_tip is added in element options [X] sanitized.
+		 * desc_tip is added in element options [X] sanitized to fasle.
```

Comment:
Please fix indentation. also "false"  at the end has spelling mistake.

Reply:
this line should not be indented could however have a new line above  
spilling will be corrected